### PR TITLE
Some typos and grammatical errors

### PR DESCRIPTION
**Pull request by kunakl07**
I have fixed some typing and some grammatical mistakes.

### CODE REVIEW

File: doc/api_changes.rst [hunk 1]
```diff
@@ -170,7 +170,7 @@ If you want to assure that your code runs in exactly the same manner as before,
 - All available actors are in ``dipy.viz.actor`` instead of ``dipy.fvtk.actor``.
 - UI elements are available in ``dipy.viz.ui``.
 
-``dipy.viz`` depends on FURY package. To get more informations about FURY, go to https://fury.gl
+``dipy.viz`` depends on the FURY package. To get more pieces of information about FURY, go to https://fury.gl
```
Comment (arokem) [inline doc/api_changes.rst]: How about this instead?

```suggestion
``dipy.viz`` depends on the FURY package. To learn more about FURY, go to https://fury.gl
```

File: doc/api_changes.rst [hunk 2]
```diff
@@ -181,7 +181,7 @@ DIPY 0.14 Changes
 ``dipy.io.trackvis`` module is deprecated. Use ``dipy.io.streamline`` instead. Furthermore,
 ``load_trk`` and ``save_trk`` from ``dipy.io.streamline`` is highly recommended for managing streamlines.
 When you create streamlines, you should use ``from dipy.tracking.streamlines import Streamlines``. This new
-object uses much less memory and it is easier to process.
+the object uses much less memory and it is easier to process.
```
Comment (arokem) [inline doc/api_changes.rst]: Please revert this change:

```suggestion
object uses much less memory and it is easier to process.
```

File: doc/api_changes.rst [hunk 3]
```diff
@@ -243,12 +243,12 @@ DIPY 0.10 Changes
...
 
 ``fvtk.slicer`` input parameters have changed. Now the slicer function is
-more powerfull and supports RGB images too. See tutorial ``viz_slice.py`` for
+more powerful and supports RGB images too. See tutorial ``viz_slice.py`` for
 more information.
 
 **Interpolation**
 The default behavior of the function `core.sphere.interp_rbf` has changed.
-The default smoothing parameter is now set to 0.1 (previously 0). In addition,
+The default smoothing parameter is now set to 0.1 (previously 0). Besides,
```
Comment (arokem) [inline doc/api_changes.rst]: Please revert:

```suggestion
The default smoothing parameter is now set to 0.1 (previously 0). In addition,
```

File: doc/api_changes.rst (hunk 1 above)
Comment (kunakl07) [reply doc/api_changes.rst]: Yeah! That's even better

File: doc/api_changes.rst (hunk 2 above)
Comment (kunakl07) [reply doc/api_changes.rst]: Thanks for pointing out.
Even I don't remember why did I put 'the' here when it made no sense

File: doc/api_changes.rst (hunk 3 above)
Comment (kunakl07) [reply doc/api_changes.rst]: Sure

**Comment by skoudoro** [comment]
@arokem, What is the status of this PR, is it good for you?

File: doc/api_changes.rst [hunk 4]
```diff
@@ -170,7 +170,7 @@ If you want to assure that your code runs in exactly the same manner as before,
 - All available actors are in ``dipy.viz.actor`` instead of ``dipy.fvtk.actor``.
 - UI elements are available in ``dipy.viz.ui``.
 
-``dipy.viz`` depends on FURY package. To get more informations about FURY, go to https://fury.gl
+``dipy.viz`` depends on the FURY package. To get learn about FURY, go to https://fury.gl
```
Comment (arokem) [inline doc/api_changes.rst]:
```suggestion
``dipy.viz`` depends on the FURY package. To learn more about FURY, go to https://fury.gl
```

**Comment by arokem** [comment]
One small additional suggestion from me.

**Comment by kunakl07** [comment]
> One small additional suggestion from me.

Yea sure. Thanks
